docs/pages/guides/testing-rsc.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Testing React Server Components
+sidebar_title: Testing RSC
+description: Learn about writing unit tests for React and React Native Server Components in Expo.
```

Comment:
The description mentions "React" here. Is it different from using Unit testing in general (the instructions we mention here: https://docs.expo.dev/develop/unit-testing/)?

Reply:
It's like unit testing if the tests ran on an iOS device instead of in the terminal.